src/XIVLauncher.Core/Components/MainPage/MainPage.cs

```diff
@@ -698,6 +698,8 @@ public async Task<Process> StartGameAndAddon(Launcher.LoginResult loginResult, b
             throw new NotImplementedException();
         }
 
+        App.Settings.AdditionalArgs = $"UserPath={Program.CompatibilityTools.UnixToWinePath(App.Settings.GameConfigPath.FullName)}";
```

Comment:
Doing this, you override the existing AdditionalArgs setting. Users might have something in there that they want to use. You could:

```csharp
var args = $"UserPath=[...]"
if (App.Settings.AdditionalArgs != null)
	args += $" {App.Settings.AdditionalArgs}";

// use "args" to start game instead of App.Settings.AdditionalArgs

Reply:
Thanks, I've updated the code to append the `UserPath` arg to the end of `AdditionalArgs` and then I trim any leading/trailing whitespace.